fbw-a32nx/src/systems/instruments/src/PFD/AttitudeIndicatorHorizon.tsx

```diff
@@ -382,15 +382,15 @@ class RadioAltAndDH extends DisplayComponent<{ bus: EventBus, filteredRadioAltit
                 const chosenTransalt = this.fmgcFlightPhase <= 3 ? this.transAlt : this.transAltAppr;
                 const belowTransitionAltitude = chosenTransalt !== 0 && (!this.altitude.isNoComputedData() && !this.altitude.isNoComputedData()) && this.altitude.value < chosenTransalt;
                 let size = 'FontLarge';
-                const DHValid = this.dh >= 0;
+                const DHValid = this.dh.value >= 0 && !this.dh.isNoComputedData();
```

Comment:
You should also check for failure warning
```suggestion
                const DHValid = this.dh.value >= 0 && (!this.dh.isNoComputedData() || !this.dh.isFailureWarning());
```

Reply:
`!this.dh.isFailureWarning()`

will always be true when it sends NoComputedData. So the correct check would be something like this `(this.dh.ssm > Arinc429SignStatusMatrix.NoComputedData)`

Reply:
Sorry, that was a typo - I mean `(!this.dh.isNoComputedData() && !this.dh.isFailureWarning());` of course.

> (this.dh.ssm > Arinc429SignStatusMatrix.NoComputedData)
I think this is hard to read, because you have to know that FW is above NCD.